src/server/game/Entities/GameObject/GameObjectData.h

```diff
@@ -669,4 +672,33 @@ struct GameObjectData : public SpawnData
     uint8 artKit = 0;
 };
 
+enum class GameObjectActions : uint32
+{
+    // Name from client executable      // Comments
```

Comment:
If you don't mind, please indent this line for it to be aligned with the comments on the subsequent lines. It's the first column of the comments that's meant to be "Name from client executable".